docs/contents.md

```diff
@@ -7,8 +7,8 @@
 
 - Ember
```

Comment:
>Since Glint currently only works with TypeScript

Glint actually works with JS (#199), it's just not documented yet (#208).

Reply:
> Glint actually works with JS (#199), it's just not documented yet (#208).

Awesome! I do still think a link to e-cli-ts would be useful _somewhere_ since people frequently use them together. My concern is that someone might assume they only need Glint since the overview says "Glint's CLI provides a template-aware tool for performing end-to-end TypeScript typechecking"